README.md

```diff
@@ -30,6 +30,9 @@ uses Gradle properties. It's generally recommended to set them in your `gradle.p
 file.
 
 ```properties
+SONATYPE_HOST=DEFAULT
```

Comment:
I really don't like that you have to specify this, why can't we do `default` always and only if you want to change it, you have to declare it

Reply:
In general there are 2 reasons for this
- I think it's generally safer to not add something by default, this way you won't accidentally publish something internal by just running `publish`
- From the plugin perspective we can avoid a lot of edge cases by having less  `afterEvaluate` blocks, we won't be able to get rid of all of them (like when detecting which plugins are applied to the build) but at least some. 

With properties we can still offer easy, low overhead configuration without the cost of afterEvaluate. If you want to publish to maven central you need to add the other properties anyways so it doesn't really increase the setup cost. 

The reason for not using `default` as the value when no property is present is that it won't work well with the DSL. We would then call `publishToMavenCentral(DEFAULT)` immediately and users wouldn't be able to call `publishToMavenCentral(S01)` if the prefer to use the DSL over properties.